src/autoTune.ml

```diff
@@ -45,20 +45,22 @@ class functionVisitor(calling, calledBy, argLists, dynamicallyCalled) = object
 end
 
 exception Found
-class findAllocsNotInLoops = object
+class findAllocsInLoops = object
   inherit nopCilVisitor
 
+  val mutable inloop = false
+
   method! vstmt stmt =
     match stmt.skind with
-    | Loop _ -> SkipChildren
+    | Loop _ -> inloop <- true; DoChildren
```

Comment:
`inloop` also needs to be reset after the children are processed (like how loop nesting counter is decremeted). Otherwise allocations after a loop will also have `inloop` `true`.